posthog/settings/overloads.py

```diff
@@ -0,0 +1,46 @@
+# Imported before anything else to overwrite env vars!
```

Comment:
I'd suggest `overrides` instead of `overloads` as the file name – reading `overloads` I thought that it's something about operators instead of plain overwriting of env vars.